### ISSUE TITLE
Color Contrast ratio of focus present on mycodespace1 list item is less than 3:1.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/vscode#105722]
["Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yWCAG2.1; #A11yTCS; #Win32; #DesktopApp; #A11y_VSCode;#WCAG1.4.11; #A11ySev2;#CAITestPass_May2020;#Benchmark;#Visual Studio Code;

**Environment Details:​**
Application Name: VS Codespace
Operating System: Win10​
Version: 1.47.0-insider (user setup)
Commit: d5e9aa0227e057a60c82568bf31c04730dc15dcd
Date: 2020-07-08T10:52:52.439Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041

**Additional Details:**
MAS References:WCAG1.4.11

**Repro Steps:​**
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Codespace"
3. Click on "Visual Studio codespace".
4.Launch AI4D.
5.Navigate up to remote explorer section and click on it.
6. Select the list item under Cloud Hosted codespace and Verify the color contrast of the keyboard focus.

**Actual:**

Color Contrast ratio of keyboard focus present on mycodespace1 list item is less than 3:1.

**Same issue is also observed in "Text Box Family" screen in which focus is not move to "Clear All" and "Show Password" button.**

**Expected:**

Closed Tab button and Clear all buttons should be in keyboard focus order for closing tab.

**User Impact:**

Keyboard users will not be able to closed tab with the help of keyboard key.

**Attachment link for Reference:**
[#105722.zip](https://github.com/microsoft/vscode/files/5149863/105722.zip)

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!
